the-blue-alliance-ios/View Controllers/Districts/DistrictRankingsViewController.swift

```diff
@@ -179,16 +179,10 @@ extension DistrictRankingsViewController: Refreshable {
             }
 
             self.persistentContainer.performBackgroundTask({ (backgroundContext) in
-                let backgroundDistrict = backgroundContext.object(with: self.district.objectID) as! District
```

Comment:
This code needs to come back - our controller updating in the background (for now) seems like it should be responsible for grabbing a background object. I might have a change of heart at some point and decide that the callers to model methods shouldn't have to bear the burden of making sure they're thread safe, but that's not yet